src/Compilers/CSharp/Test/Semantic/Diagnostics/GetDiagnosticsTests.cs

```diff
@@ -394,5 +394,43 @@ public void CompilingCodeWithSameParseOptionsInMultipleSyntaxTreesShouldReportOn
             Assert.True(diagnostics[0].Location.SourceTree.Equals(syntaxTree1));
             Assert.True(diagnostics[1].Location.SourceTree.Equals(syntaxTree2));
         }
+
+        [Fact]
+        [WorkItem(11497, "https://github.com/dotnet/roslyn/issues/11497")]
+        public void ConsistentErrorMessageWhenProvidingNullKeyFile()
```

Comment:
I think this fix is related to signing. We are making changes around options that affect signing exclusively. Both success and error cases should be tested.